docs/getting_support.rst

```diff
@@ -3,45 +3,435 @@
+management tool and email client.
+
+If you do not know how to do things like import and export GPG keys,
+the encrypted email workflow is not for you. We recommend you stick to
+the :ref:`web-based workflow <using_redmine>`.
+
+Providing your public key
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Start by navigating to
+`support.freedom.press/pgp <https://support.freedom.press/pgp>`_ or
+click on the "PGP" menu option in the upper right corner of the support window.
+
+|PGPMenu|
+
+In the PGP management window you will see two columns: one for your
+public key on the left, and one with the server's public key on the
+right.
+
+|PGPUpload|
+
+To provide your public key, start by exporting an ASCII-armored copy of
+your public key. Paste your armored public key into the empty text box
+on the left and hit **Save**.
+
+|PGPSave|
+
+Importing the server's public key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now, import the public key for our Redmine server into your local GPG
+keyring. It is available on the right hand side of the page in the
+column with the heading "Redmine Server (support@freedom.press)", in the
+text box labeled "Public PGP key".
+
+Select the entire public key and copy it to your clipboard. Import the
+copied public key into your local GPG keyring.
+
+You can `download the key <https://freedom.press/documents/37/redmine_public_key.asc>`_ 
+or fetch it from a `keyserver <https://keys.openpgp.org/vks/v1/by-fingerprint/D0E0B2F2B71BA4E48278037D9EA33029E9FBBA2E>`_ (fingerprint:
+``D0E0B2F2B71BA4E48278037D9EA33029E9FBBA2E``).
+
+
+.. todo:: Add section on testing encrypted email after setting it up
+
+.. |PGPMenu| image:: images/support/pgp_menu.png
+.. |PGPUpload| image:: images/support/pgp_upload.png
+.. |PGPSave| image:: images/support/pgp_save.png
+
+Creating a new issue via email
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sending an email to support@freedom.press will automatically create a
+new issue in your project, using the subject line of the email for the
+Subject and the body of the email for the Description.
+
+Always sign and encrypt your emails
+-----------------------------------
+
+To maintain the security of conversations around support requests, you
+should always sign and encrypt your email to support@freedom.press if
+you have a Redmine account.
+
+Our server is configured to reject emails that do not have a valid
+signature. We cannot reject unencrypted emails due to a limitation in
+the Redmine-OpenPGP plugin we are using, but we are logging them and
+will harangue you if you send us unencrypted email.
+
+PGP/MIME vs. inline PGP
 -----------------------
+
+When you're replying to Redmine via email, you **must use PGP/MIME**
+instead of inline PGP. The default settings for the Enigmail plugin in
+Thunderbird should work.
+
+Check your mail client and GPG integration tool's settings to ensure you
+are using PGP/MIME. One notable exception to this is Mailvelope, which
+cannot send PGP/MIME (you shouldn't be using Mailvelope anyway).
```

Comment:
I think we should either have an explicit "don't use Mailvelope" or remove this aside, it doesn't seem helpful as is.

Reply:
I think it's probably worth keeping the warning in some form, "Note that Mailvelope does not support PGP/Mime" (AFAICT it still doesn't: https://github.com/mailvelope/mailvelope/issues/41), I just thought the admonishment "you shouldn't be using it anyways" was out of place.